src/SFA.DAS.EmploymentCheck.Functions/AzureFunctions/Triggers/ResetEmploymentChecksMessageSentDateOrchestratorHttpTrigger.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.EmploymentCheck.Functions.AzureFunctions.Orchestrators;
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.EmploymentCheck.Functions.AzureFunctions.Triggers
+{
+    public static class ResetEmploymentChecksMessageSentDateOrchestratorHttpTrigger
+    {
+        private const string InstancePrefix = "ResetEmploymentCheckMessageSentDate-";
+
+        // Call with Postman POST request
+        // By CorrelationId: http://localhost:7071/api/orchestrators/ResetEmploymentChecksMessageSentDateOrchestratorHttpTrigger?CorrelationId=E269AE35-5A56-4DC8-A478-170E88280C31
+        // By MessageSentDate range: http://localhost:7071/api/orchestrators/ResetEmploymentChecksMessageSentDateOrchestratorHttpTrigger?MessageSentFromDate=2022-03-23&MessageSentToDate=2022-03-25)
```

Comment:
This comment is useful info but it should be documented somewhere out of code

Reply:
Removed comment.